Workbooks/WorkbooksMetadata.json

```diff
@@ -674,5 +674,57 @@
     "templateRelativePath": "Perimeter81OverviewWorkbook.json",
     "subtitle": "",
     "provider": "Perimeter 81"
+  },
+  {
+    "workbookKey": "SymantecProxySGWorkbook",
```

Comment:
SymantecProxySG doesn;t have a logo file

Reply:
unable to find a logo for product